### DISCUSSION TITLE
Sooo wait, I can make minecraft mods in Lua now?

### DISCUSSION THREAD

**Discussion by CitizenFoffie** [moongardenmods/allium#36] [General]

**Comment by hugeblank** [comment]
Yes, if you're using Fabric, and know how to build and install this mod from the source (and install fabric API if you want to add items and stuff). As soon as this mod is ready for release you'll be able to easily download and install it from Modrinth.

Also since we don't really have good documentation you'd be out of luck on help unless you knew how to read both the games source code, and this mods source code. The latter, however, is something I wouldn't want to force on anyone. I'm working on getting some semblance of documentation together. Progress has been slow as I've been quite busy this summer. 

For future reference:
a) Please if you're going to report an issue of this nature, add a description. I get that you were just asking a singular question, but surely you'd have more if I just answered with a flat out "yes." Try to be preemptive, it could save both of us some time.
b) This one is on me, but a question like this is better suited on a forum, or a discord server. GitHub issues is a last resort.

With that said, feel free to join us (allium contributors & I) over at my discord. I didn't really think allium was significant enough to have its own discord, so it's been merged with the rest of what I do (streaming, gaming, other general nerdy stuff). This link will drop you right into ~~#allium~~, although I do recommend that you read the rules first. I'll probably add that link to the README at some point here. Regardless, any further questions you may have can be dropped in there!

Thanks for your interest!

**Comment by hugeblank** [comment]
Update! This is an issue that was made prior to being converted to a discussion by yours truly. I had totally forgot that Github had this feature, and now that I know it does I'm going to try using this as a primary platform for Allium discussion.

**Comment by cad3264** [comment]
Cannot wait for it to release on modrinth! i wont have to use something like Mcreator to create mods.

**Comment by hugeblank** [comment]
Update, Allium has been released on modrinth, and now has [its own discord server](https://discord.gg/rWSaP222G9).
